pym/bob/cmds/build.py

```diff
@@ -976,15 +1012,29 @@ def _cookPackageStep(self, packageStep, checkoutOnly, depth):
                     self._runShell(packageStep, "package")
                     packageHash = hashWorkspace(packageStep)
                     audit = self._generateAudit(packageStep, depth, packageHash)
+                    # copy to shared location if possible
+                    if self.__installShared and packageStep.isShared():
+                        sharedpackagepath = os.path.join(self.__sharedFolder, asHexStr(packageBuildId))
+                        print(colorize("     Install in shared location {}".format(sharedpackagepath), "32"))
+                        if os.path.exists(sharedpackagepath):
+                            shutil.rmtree(sharedpackagepath)
```

Comment:
The installation is racy if two instances run in parallel. It should be done with a temporary directory and an atomic rename of the whole folder like in `jenkins.py`.